apps/cowswap-frontend/src/modules/wallet/api/pure/Web3StatusInner/styled.ts

```diff
@@ -92,7 +92,7 @@ export const Wrapper = styled.div`
   ${({ theme }) => theme.mediaWidth.upToMedium`
     width: auto;
     height: 100%;
-    margin: 0 0 0 auto;
```

Comment:
Is this equivalent? before we had the margin define always, now is `upToMedium` ?

Reply:
Great question!
Actually, the value of margin was overridden here:
https://github.com/cowprotocol/cowswap/pull/3108/commits/07f4c8d23318f57b02d5ac6b3ebd26529866fd5f

I've checked all places where the component is used and it's displayed well there